Tests/ArcGISToolkitTests/BookmarksTests.swift

```diff
@@ -72,7 +72,7 @@ final class BookmarksTests: XCTestCase {
         let firstBookmark = try XCTUnwrap(map.bookmarks.first)
         bookmarks.selectBookmark(firstBookmark)
         XCTAssertFalse(_isPresented)
-        wait(for: [expectation], timeout: 1.0)
+        await fulfillment(of: [expectation], timeout: 1.0)
```

Comment:
FYI: This API was [introduced in Xcode 14.3](https://developer.apple.com/documentation/xcode-release-notes/xcode-14_3-release-notes#Testing), which is above the minimum Xcode version for this project.

Reply:
@philium Thanks for catching that. This will have to wait then.

Reply:
Since Xcode 14.3 shipped with a newer version of the Swift compiler, you could use conditional compilation:

```swift
#if compiler(>=5.8)
        await fulfillment(of: [expectation], timeout: 1.0)
#else
        wait(for: [expectation], timeout: 1.0)
#endif
```

However, the whole reason for the new API is that the old API could cause a test to deadlock when used in concurrent test methods. So it might be better to avoid using expectations in this method for now.